### DISCUSSION TITLE
Inconsistency in Material You color palette across Florisboard and Android versions

### DISCUSSION THREAD

**Discussion by itsmartashub** [florisboard/florisboard#2409] [Q&A] [answered]
Hey, greetings. Does anyone happen to know if the latest update of Florisboard also updated any Material You-related dependencies?

The dynamic color palette `MY` version that Florisboard uses have changed with the latest `v0.4.0-beta01` update. Also, there are some [noticed color issues](https://github.com/itsmartashub/floristyle/issues/7) in **Android 14** vs **Android 13** 

I'm asking because I noticed that color palettes have also been changed in some other Material You Android applications that recently received updates.

I would like to fix [floristyle](https://github.com/itsmartashub/floristyle) themes that seem broken in the latest [`v0.4.0-beta01`](https://github.com/florisboard/florisboard/releases/tag/v0.4.0-beta01) version, especially for **A14** devices but I don't see the corresponding shades in this dynamic color palette. I'm not sure what or where is the issue actually.

### ADDITIONAL QUESTION:
I noticed a new metadata named `isMaterialYouAware` which is set to `false` everywhere by default. However, I don't see this option editable in the UI, so I'm not sure what it refers to or if it's necessary for theme creators to know about it?

**Comment by jindalpratik** [comment]
Might be the same issue as mentioned in this commit for another app https://github.com/jarnedemeulemeester/findroid/commit/0a1582f47b33147c6c797c76f674bdedad4a4909.

**Comment by patrickgold** [comment, marked as answer]
Actively nothing has been changed directly, however the underlying material3 dependency has been upgraded from 1.1.0 to 1.2.0. The changelog can be found [here](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.2.0), maybe it contains some changes relevant to how you apply colors in your theme.

**Comment by itsmartashub** [comment]
Thank you for the informative reference. Yes, reading the Material3 blog, version `v1.2.0` brings [updates regarding colors](https://material.io/blog/material-3-compose-1-2), mainly focusing on surface colors.

However, it seems that these colors currently don't exist in the available `MD3` color roles in Florisboard, so unfortunately nothing I can do for now.

If time allows, would you consider expanding the `MD3` color roles in some future?

**Reply by patrickgold** [reply]
yeah sure can you create a new issue for this and reference the newly added colors? Also if possible describe a way for me to easily test this.

**Reply by itsmartashub** [reply]
Thank you so much. Of course, I'll try to visually represent what it's about without complicating things too much.
